pkg/didcomm/packager/packager.go

```diff
@@ -52,11 +55,13 @@ func New(ctx Provider) (*Packager, error) {
 
 	basePackager.primaryPacker = ctx.PrimaryPacker()
 	if basePackager.primaryPacker == nil {
-		return nil, fmt.Errorf("need primary primaryPacker to initialize packager")
+		return nil, fmt.Errorf("need primary packer to initialize packager")
 	}
 
 	basePackager.addPacker(basePackager.primaryPacker)
 
+	basePackager.connectionStore = ctx.DIDConnectionStore()
```

Comment:
it seems odd to see this direct mutation of the field.

why isn't packager constructed with the store?